report/src/statChart.ts

```diff
@@ -0,0 +1,212 @@
+import { QuartileCandlestick, Scatter } from "@hpcc-js/chart";
+import { format as d3Format, HTMLWidget, Palette } from "@hpcc-js/common";
+
+const rainbow = Palette.rainbow("Blues");
+const palette = Palette.ordinal("Quartile", [rainbow(100, 0, 100), rainbow(50, 0, 100), rainbow(50, 0, 100), rainbow(75, 0, 100)]);
+palette("Std. Dev.");
+palette("MinMax");
+palette("25%");
+palette("50%");
+
+type QuertilesType = [number, number, number, number, number];
+type Mode = "min_max" | "25_75" | "normal";
+type Tick = { label: string, value: number };
+type Ticks = Tick[];
+type AxisTick = { label: string, value: string };
+type AxisTicks = AxisTick[];
+
+const CandleHeight = 20;    //  Pixels
+const DomainPadding = 10;    // Percentage
```

Comment:
Should use UPPER_SNAKE_CASE for global constants

Reply:
Done